src/content/docs/it/concepts/islands.mdx

```diff
@@ -0,0 +1,66 @@
+
+## Come funzionano le isole in Astro?
+
+**Astro genera ogni sito web senza alcun JavaScript lato client, per impostazione predefinita.** Utilizza un componente UI frontend costruito con [React](https://react.dev/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) o [Lit](https://lit.dev/), e Astro lo renderizzerà automaticamente in HTML in anticipo per poi rimuovere tutto il JavaScript. Questo mantiene ogni sito veloce per impostazione predefinita, eliminando tutto il JavaScript inutilizzato dalla pagina.
+
+```astro title="src/pages/index.astro"
+---
+// Esempio: Usa un componente React statico sulla pagina, senza JavaScript.
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<!-- 100% HTML, Zero JavaScript caricato sulla pagina! -->
+<MyReactComponent />
+```
+
+Ma talvolta è necessario del JavaScript lato client per creare un'interfaccia utente interattiva. Invece di costringere l'intera pagina a diventare un'applicazione JavaScript simile ad un'applicazione a pagina singola (SPA), Astro ti chiede di creare un'isola.
+
+```astro title="src/pages/index.astro" ins="client:load"
+---
+// Esempio: Usa un componente React dinamico sulla pagina.
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<!-- Questo componente ora è interattivo sulla pagina!
+     Il resto del tuo sito web rimane statico e senza JS. -->
+<MyReactComponent client:load />
+```
+
+Con le Isole Astro, la stragrande maggioranza del tuo sito rimane in HTML e CSS. Nell'esempio sopra, hai appena aggiunto uns singola **isola di interattività**, isolata dal resto della pagina, senza modificare anche il resto della pagina.
+
+## Quali sono i vantaggi delle isole?
+
+Il vantaggio più evidente nell'utilizzo delle Isole Astro è la performance: la maggior parte del tuo sito web viene convertito in HTML statico veloce e JavaScript viene caricato solo per i singoli componenti che ne hanno bisogno. JavaScript è uno degli asset più lenti da caricare per byte, quindi ogni byte conta.
+
+Un altro vantaggio è il caricamento parallelo. Nell'illustrazione di esempio sopra, l'isola "carosello di immagini" a bassa priorità non deve bloccare l'isola "intestazione" ad alta priorità. Le due vengono caricate in parallelo e idratate in isolamento, il che significa che l'intestazione diventa immediatamente interattiva senza dover aspettare il carosello più pesante più in basso nella pagina.
+
+Ancora meglio, puoi dire ad Astro esattamente come e quando renderizzare ciascun componente. Se quel carosello di immagini è davvero costoso da caricare, puoi aggiungere una [direttiva client](/it/reference/directives-reference/#client-directives) speciale che indica ad Astro di caricare il carosello solo quando diventa visibile sulla pagina. Se l'utente non lo vede mai, non viene mai caricato.
```

Comment:
Just a suggestion: maybe "pesante" can be used instead of "costoso", it's quite common to use "pesante" when talking about download size.

Reply:
I applied the fixes!